redirects.js

```diff
@@ -27,8 +27,8 @@ function redirects() {
       permanent: false,
     },
     {
-      source: "/contracts/publish",
-      destination: "/contracts/release",
+      source: "/contracts/release",
+      destination: "/contracts/publish",
       permanent: false,
     },
```

Comment:
we need a redirect here for `/release` to `/publish` for the landing

Reply:
Also another one for any match, so for example:

https://thirdweb-www-git-sid-release-to-publish.thirdweb-preview.com/contracts/release/QmY3ZMRSQUN3WGhdqt7jCdviA6Jj16b8iN1C954pZSDpGX%2F0

would bring me to the correct url

we can do it by adding this:

```
{
  source: "/contracts/release/:path*",
  destination: "/contracts/publish/:path*",
  permanent: false,
},
```